Makefile

```diff
@@ -37,7 +37,7 @@ $(BUILDDIR)/apple/Tun2socks.xcframework: $(BUILDDIR)/ios/Tun2socks.xcframework $
 XGO=$(GOBIN)/xgo
 TUN2SOCKS_VERSION=v1.16.11
 XGO_LDFLAGS='-w -X main.version=$(TUN2SOCKS_VERSION)'
-ELECTRON_PKG="./client/src/tun2socks/outline/electron"
+ELECTRON_PKG="./client/src/backend/outline/electron"
```

Comment:
Please double confirm this won't change the final binary name (it should be `tun2socks` for Linux and `tun2socks.exe` for Windows).

Or maybe we should actually update it to `backend` and `backend.exe`? It's up to you.